BizTravel360/src/main/java/com/travel/BizTravel360/Person/Person.java

```diff
@@ -0,0 +1,61 @@
+package com.travel.BizTravel360.Person;
+
+import java.util.Objects;
+
+public class Person {
+    private final int id;
```

Comment:
```suggestion
    private long personId;
```
preferowanym typem zmiennej do trzymania ID jest long, bardziej sugestywną nazwą będzie personId, final nie jest konieczny, ja się nie spotkałem z takimi oznaczaniem id